docs/runtimes/application.ipynb

```diff
@@ -135,6 +135,40 @@
     ")"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        # The name of the API gateway, defaults to -
</pre><p>the default name is used only when deploying with default api gateway. when you use <code>create_api_gateway</code> directly, you must provide a name </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/6275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Fixed in both places